aws/resource_aws_ssm_association.go

```diff
@@ -38,6 +39,30 @@ func resourceAwsSsmAssociation() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"max_concurrency": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, es []error) {
```

Comment:
This can be simplified with [`validation.StringMatch()`](https://godoc.org/github.com/hashicorp/terraform/helper/validation#StringMatch). It looks like this regular expression might also be slightly different than the max errors one according to the [SSM API Reference](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_CreateAssociation.html#systemsmanager-CreateAssociation-request-MaxConcurrency) 👍 

e.g.

```go
ValidateFunc: validation.StringMatch(regexp.MustCompile(`^([1-9][0-9]*|[1-9][0-9]%|[1-9]%|100%)$`), "must be a valid number (e.g. 10) or percentage including the percent sign (e.g. 10%)"),
```